### PR TITLE
Finish the token related API for alpha

### PR DESCRIPTION
**Pull request by heliuchuan**
### Description
Adds getCurrentTokenOwnerData, getTokenActivities, getOwnedToken queries to the token client.  Adds mintTokenTransaction and a simple example.

### Test Plan
pnpm test

### CODE REVIEW

File: tests/e2e/api/token.test.ts [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
+    const config = new AptosConfig({ network: Network.MAINNET });
+    const aptos = new Aptos(config);
+    const tokenAddress = "0x0cb098d8f875f38dcb4109e2638e3e24055a585ed2143e9ba76e002fea303795";
+    const tokenData = await aptos.getTokenData({ tokenAddress });
+    expect(tokenData).toHaveProperty("description");
+    expect(tokenData).toHaveProperty("current_collection");
+    expect(tokenData).toHaveProperty("token_data_id");
+    expect(tokenData.current_collection).toHaveProperty("collection_id");
+    expect(tokenData.current_collection).toHaveProperty("collection_name");
+    expect(tokenData.current_collection).toHaveProperty("creator_address");
```
[the remark this answers is not in the document]
Comment (heliuchuan) [reply tests/e2e/api/token.test.ts:18]: yeah but not token yet, which I'm just wrapping up

File: src/api/token.ts [hunk 2]
```diff
@@ -15,6 +31,28 @@ export class Token {
...
+   *
+   * @param args.creator the creator of the collection
+   * @param args.collection the name of the collection the token belongs to
+   * @param args.description the description of the token
+   * @param args.name the name of the token
+   * @param args.uri the URI to additional info about the token
+   *
+   * @returns GetTokenDataResponse containing relevant data to the token.
+   */
+  async mintTokenTransaction(args: {
```
Comment (heliuchuan) [inline src/api/token.ts:45]: mintTokenTransaction or mintTransaction?

Comment (gregnazario) [reply src/api/token.ts:45]: Because they all roll up, they will need Token in the name.

**Comment by heliuchuan** [comment]
Yeah, I was planning to do that in follow up,the scope of this PR has already grown quite a bit.

**Pull request merged**